package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "author": "<add your company or personal name here>",
   "scripts": {
+    "gen": "generate-it --yes --mocked --template https://github.com/acrontum/generate-it-typescript-server ./api.yml",
```

Comment:
this should not be here

Reply:
removed - was a hacky solution to the core issue